lib/opencomponents.rb

```diff
@@ -10,17 +10,20 @@ module OpenComponents
   # Public: Default OC registry URL (http://localhost:3030).
   DEFAULT_REGISTRY = 'http://localhost:3030'
 
+  # Public: Default request timeout in seconds (5)
+  DEFAULT_TIMEOUT = 5
+
   # Internal: Custom exception class to raise in the event a component cannot be
   #   found in the registry.
   ComponentNotFound = Class.new(RuntimeError)
 
   # Internal: Custom exception class to raise for response timeouts.
-  RegistryTimeout = Class.new(RestClient::RequestTimeout)
+  RegistryTimeout = Class.new(RuntimeError)
```

Comment:
`StandardError` is overly broad. It's pretty much akin to just subclassing `Exception`.